code_search/docker/ks/submit_code_embeddings_job.sh

```diff
@@ -21,37 +21,45 @@ workerMachineType=n1-highcpu-32
 
 usage() {
 	echo "Usage: submit_code_embeddings_job.sh
-	--workflowId=<workflow id invoking the container>
-	--modelDir=<directory contains the model>
+	--cluster=<cluster to deploy job to>
 	--dataDir=<data dir>
-	--functionEmbeddingsDir=<output function embedding dir>
-	--tokenPairsBQTable=<input token pairs BQ table>
+	--failedTokenizeBQTable=<output failed tokenize BQ table>
 	--functionEmbeddingsBQTable=<output function embedding BQ table>
+	--functionEmbeddingsDir=<output function embedding dir>
+	--modelDir=<directory contains the model>
+	--namespace=<kubernetes namespace>
 	--numWorkers=<num of workers>
 	--project=<project>
+	--timeout=<timeout>
+	--tokenPairsBQTable=<output token pairs BQ table>
+	--vocabularyFile=<the pre-generated vocabulary file>
 	--workerMachineType=<worker machine type>
-	--workingDir=<working dir>
-	--cluster=<cluster to deploy job to>
-    --namespace=<kubernetes namespace>"
+	--workflowId=<workflow id invoking the container>
+	--workingDir=<working dir>"
 }
 
 # List of required parameters
-names=(dataDir modelDir functionEmbeddingsDir tokenPairsBQTable functionEmbeddingsBQTable workingDir workflowId cluster namespace)
+names=(cluster dataDir failedTokenizeBQTable functionEmbeddingsBQTable functionEmbeddingsDir modelDir namespace project tokenPairsBQTable workflowId workingDir)
 
 source "${DIR}/parse_arguments.sh"
 source "${DIR}/initialize_kubectl.sh"
 
+gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
```

Comment:
Why do we need to copy the vocab file? Could we instead modify the Dataflow pipeline so that change isn't necessary?

Reply:
done. i instead pass an existing directory with vocabulary file as dataDir